CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 * Adds `allow_private_set` configuration for the `private_outlet` rule.  
   [Rohan Dhaimade](https://github.com/HaloZero)
+  
+* Added test to correctable rules to check `swiftlint-disable` before auto-correcting file.  
```

Comment:
Also, I'd call this a bug fix and the fact that you added tests is an implementation detail. The entry should read something like this IMO:

> Correctable rules no longer apply corrections if the rule is locally disabled.